code/include/env/mcountinhibit.h

```diff
@@ -0,0 +1,24 @@
+#ifndef RISCV_MCOUNTINHIBIT_H
+#define RISCV_MCOUNTINHIBIT_H
+
+#define MCOUNTINHIBIT_CY_SHIFT    0
+#define MCOUNTINHIBIT_0_SHIFT     1
```

Comment:
This is a reserved field with value zero, so should be left out i.e. not have any defines for. Please also fix for the mask and remove.